pom.xml

```diff
@@ -43,8 +43,11 @@ limitations under the License.
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.starrocks</groupId>
     <artifactId>flink-connector-starrocks</artifactId>
-    <version>1.2.3_flink-1.15</version>
+    <version>${srfc.version}_flink-1.15</version>
```

Comment:
You can add a more clear commit message about the problem, solution and results

Reply:
flink state should contains srfc.version, flag the struct version